### PR TITLE
Refactor AFK function + Support for Logged out + Fix Locale message

### PR DESCRIPTION
**Pull request by S33G**
- Much less spagetti code
- Reinstates locale message as it was missing
- Includes AFK for logged out users (ie. qb-multicharacter)
- Adds config option for enable/disable
- Changed notification duration to avoid overlapping notifications

Fixes:
https://github.com/qbcore-framework/qb-smallresources/issues/281

Duplicates:
https://github.com/qbcore-framework/qb-smallresources/pull/331

### CODE REVIEW

**Review by r0adra93 — APPROVED** [review]
LGTM

**Comment by zwkf** [comment]
> Hello. I went ahead and tested this to make sure it behaved how I intended. It seems to be good to me.
> 
> I still like the idea of adding a different configurable afk timer for the time before the player logs in. I just think that it should be shorter than a player who is logged in. Would be open to discussing this more and hearing your thoughts!

This is entirely fair. I feel this pr does enough right now and is still an improvement on what we have currently. 
Maybe we can introduce another config variable once this one is merged :)

**Review by Irishstevie — APPROVED** [review]
I agree just run this on a clean install and works great

File: client/afk.lua [hunk 1]
```diff
@@ -38,36 +29,50 @@ RegisterNetEvent('QBCore:Client:OnPermissionUpdate', function()
...
 
-CreateThread(function()
-    while true do
-        Wait(10000)
-        local playerPed = PlayerPedId()
-        if isLoggedIn then
-            if checkUser then
+function getTimeUnit(time)
+    return time >= 60 and 'minutes' or 'seconds'
+end
```
Comment (S33G) [inline client/afk.lua:34]: Needs a linebreak here below `end`

**Pull request closed without merging**